kijiji-query.js

```diff
@@ -42,9 +43,8 @@ var parseXML = function(xml) {
         var ad = {};
         $(item).children().each(function(i, child) {
             ad[child.name] = $(child).text();
-        });    
-    
-        ad.innerAd = {};
```

Comment:
Removing this from here breaks code that depends on the presence of the innerAd object (such as [kijiji-parser.js](https://github.com/mwpenny/kijiji-scraper/blob/master/ad-parser.js#L30)). I see your intent (if there is no innerAd information, why include it at all) but keeping this empty object allows the same code to be used with scraped _and_ unscraped innerAds with no extra checks (the string produced by kijiji-parser.js will just be blank).

The way I conceptualize it is that there should always be an innerAd, since each Kijiji ad always has that detail associated with it. However, we may or may not have actually _retrieved_ that information.